checkstyle-checks/src/main/resources/sonatype/checkstyle-configuration.xml

```diff
@@ -43,9 +43,13 @@
       <property name="option" value="TEXT"/>
       <property name="tokens" value="LITERAL_TRY, LITERAL_FINALLY, LITERAL_IF, LITERAL_ELSE, LITERAL_SWITCH"/>
     </module>
+    <module name="LeftCurly">
+      <property name="option" value="eol"/>
```

Comment:
eol is the default value for this option (cf. http://checkstyle.sourceforge.net/config_blocks.html#LeftCurly). So I doubt there is a need for this extra block. All we need is to remove `LITERAL_TRY` from the below block. Or put differently, treat it like `LITERAL_IF`

Reply:
removed in 4b16c993f817f1ae598d7d600d6561c8cd0d049d